src/changelog.ts

```diff
@@ -192,35 +195,67 @@ function stringifyLinkReferenceDefinitions(
   //
   // If there have not been any releases yet, the repo URL is used directly as
   // the link definition.
+  const orgLatestVersion = getOriginalLatestVersion();
+  const orgTagPrefix = getOriginalTagPrefix();
+  let tagPrefixToCompare = tagPrefix;
```

Comment:
It looks like we define this variable here and then change it as we loop over versions below. It feels like that could make this code more difficult to follow in the future.

Other than the changes you've made, it looks like the code responsible for defining `unreleasedLinkReferenceDefinition` and `releaseLinkReferenceDefinitions` are both fairly self-contained. What do you think about extracting these to functions? This way you can keep the definition of `tagPrefixToCompare` scoped to each of these functions.

Reply:
When we look at the function `stringifyLinkReferenceDefinitions`, the entire function is only about these two `unreleasedLinkReferenceDefinition` and `releaseLinkReferenceDefinitions`. Yeah, I agree that the function needs a refactor as it's already touching 100 lines mark. I can create a ticket for that.

Reply:
@mcmire the refactor has been done.